docs/plugins/spectate.rst

```diff
@@ -127,6 +148,16 @@ Settings
     If the ``spectate.tooltip`` overlay is enabled, toggle whether to show the
     happiness level (stress) of the dwarf in the hover panel.
 
+``tooltip-hover-stress-levels`` (default: Displeased, Content, Pleased are disabled)
+    If the ``spectate.tooltip`` overlay is enabled, toggle whether to show the
+    specific happiness level (stress) of the dwarf in the hover panel. F.e.
+    ``tooltip-hover-stress-levels 2 true`` would show Displeased emoticon.
+
+``tooltip-stress-levels``
+    Controls how happiness levels (stress) are displayed (emoticon and color).
+    F.e. ``tooltip-stress-levels 6 text XD`` will change the emoticon for
+    Ecstatic dwarves to ``XD``.
```

Comment:
the full table of default mappings for this would be useful too

Reply:
added a table and mentions of `tooltip-stress-levels` for easier discoverability of said table